src/jukebox/components/playermpd/__init__.py

```diff
@@ -359,6 +476,19 @@ def __call__(self, *args, **kwargs):
         return self._instance
 
 
+def get_music_directory(conf_file='/etc/mpd.conf'):
+    """Extract the music directory from the mpd.conf file"""
+    pattern = re.compile(r'^\s*music_directory\s*"(.*)"', re.I)
+    directory = None
+    with open(conf_file, 'r') as f:
```

Comment:
You should check if this file exists. We do have an [option](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/resources/default-settings/jukebox.default.yaml#L47) to change the location and it might cause issues!

Reply:
This is checked, where this function is used.